Hyperbolic_surface_triangulation_2/include/CGAL/Hyperbolic_surface_traits_2.h

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2024
+// INRIA Nancy (France), and Université Gustave Eiffel Marne-la-Vallee (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Vincent Despré, Loïc Dubois, Monique Teillaud
+
+// This file contains the declaration and the implementation of the class Hyperbolic_surface_traits_2
+
+#ifndef CGAL_Hyperbolic_surface_traits_2
+#define CGAL_Hyperbolic_surface_traits_2
+
+#include <CGAL/Complex_number.h>
+#include <iostream>
+
+namespace CGAL {
+
+/*
+
+template<class ComplexType>
+class Hyperbolic_point_2 {
+private:
+  ComplexType _z;
+
+public:
+  typedef ComplexType                           Complex;
+
+  Hyperbolic_point_2() { _z = Complex(); }
+  Hyperbolic_point_2(const Complex& z) { _z = z; }
```

Comment:
```suggestion
  Hyperbolic_point_2(const Complex& z) : _z(z) {}
```
The same may apply elsewhere, so please check.